.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,262 @@
+name: Build and Test Phalcon
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+  pull_request:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    name: Build Phalcon Pecl Package
+    steps:
+      - name: Set environment variables
```

Comment:
@ruudboon Could you please add a blank line before each of `- name:` to improve readability?

Reply:
will do